### ISSUE TITLE
[Help] 请问 modeling_chatglm.py 里的 `labels` 是什么数据类型？

### ISSUE THREAD

**Issue by shtdbb** [THUDM/ChatGLM-6B#1183]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

想计算一下一个例子的 loss，发现 hf 里的 modeling_chatglm.py 代码里 `ChatGLMForConditionalGeneration` 类 `forward()` 方法可以输入 `labels` 计算 loss。想问问这个 `labels` 是什么数据类型呀？是希望输出的字符串的 ids，还是 word embedding 呢？需要有什么特殊填充格式构造 `labels` 吗？谢谢！

### Steps To Reproduce

大概是 1117 行代码的 `labels`：https://huggingface.co/THUDM/visualglm-6b/blob/f4f759acde0926fefcd35e2c626e08adb452eff8/modeling_chatglm.py#L1117

`
if labels is not None:
--
  | lm_logits = lm_logits.to(torch.float32)
  |  
  | # Shift so that tokens < n predict n
  | shift_logits = lm_logits[..., :-1, :].contiguous()
  | shift_labels = labels[..., 1:].contiguous()
  | # Flatten the tokens
  | loss_fct = CrossEntropyLoss(ignore_index=-100)
  | loss = loss_fct(shift_logits.view(-1, shift_logits.size(-1)), shift_labels.view(-1))
  |  
  | lm_logits = lm_logits.to(hidden_states.dtype)
  | loss = loss.to(hidden_states.dtype)
`

### Environment

```markdown
- OS: Ubuntu 20.04
- Python: 3.8
- Transformers: 4.27.1
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by tomcat123a** [comment]
labels就是tokenize之后的text，是one hot形式的，跟embedding维度一样，是“正确答案”。参考clm causal language model的概念。
